piq/functional.py

```diff
@@ -0,0 +1,72 @@
+from typing import Tuple
+
+import torch
+
+
+def ifftshift(x: torch.Tensor):
+    """ Similar to np.fft.ifftshift but applies to PyTorch Tensors
+    """
```

Comment:
If you intend to make a single like docstring, make it actually a single line 😄 

```suggestion
    r""" Similar to np.fft.ifftshift but applies to PyTorch Tensors """
```